src/Simplex/Chat/View.hs

```diff
@@ -134,7 +134,15 @@ responseToView hu@(currentRH, user_) ChatConfig {logLevel, showReactions, showRe
   CRChatItems u _ chatItems -> ttyUser u $ concatMap (\(AChatItem _ _ chat item) -> viewChatItem chat item True ts tz <> viewItemReactions item) chatItems
   CRChatItemInfo u ci ciInfo -> ttyUser u $ viewChatItemInfo ci ciInfo tz
   CRChatItemId u itemId -> ttyUser u [plain $ maybe "no item" show itemId]
-  CRChatItemStatusUpdated u ci -> ttyUser u $ viewChatItemStatusUpdated ci ts tz testView showReceipts
+  CRChatItemsStatusesUpdated u chatItems
+    | length chatItems > 20 ->
+        if testView && showReceipts
+          then ttyUser u [sShow (length chatItems) <> " message statuses updated"]
+          else []
+    | otherwise ->
+        concatMap
+          (\ci -> ttyUser u $ viewChatItemStatusUpdated ci ts tz testView showReceipts)
+          chatItems
```

Comment:
flip the condition, to keep it flat:

| length chatItems <= 20 -> ...
| testView && showReceipts -> ...
| otherwise -> ...